tests/cephfs/cephfs_volume_management.py

```diff
@@ -0,0 +1,78 @@
+import logging
+import timeit
+import traceback
+
+from ceph.ceph import CommandFailed
+from tests.cephfs.cephfs_utils import FsUtils
+
+logger = logging.getLogger(__name__)
+log = logger
+
+
+def run(ceph_cluster, **kw):
+    try:
+        start = timeit.default_timer()
+
+        fs_util = FsUtils(ceph_cluster)
+        config = kw.get("config")
+        build = config.get("build", config.get("rhbuild"))
+        client_info, rc = fs_util.get_clients(build)
+        if rc == 0:
+            log.info("Got client info")
+        else:
+            raise CommandFailed("fetching client info failed")
+        # client1 = []
+        client1 = client_info["fuse_clients"][0]
+        rc1 = fs_util.auth_list([client1])
+        print(rc1)
+        if rc1 == 0:
+            log.info("got auth keys")
+        else:
+            raise CommandFailed("auth list failed")
+        results = []
+        tc1 = "83573446"
+        log.info("Execution of testcase %s started" % tc1)
+        log.info("Create a volume")
+        out, rc = client1.exec_command(
+            sudo=True, cmd="ceph fs volume create cephfs_new "
+        )
+        if client1.node.exit_status == 0:
+            results.append("Volume created successfully")
+
+        log.info("List the created volume")
+        out, rc = client1.exec_command(
+            sudo=True, cmd="ceph fs volume ls | grep cephfs_new "
+        )
+        if client1.node.exit_status == 0:
+            results.append("Listing of volume is successful")
+
+        log.info("Delete a volume")
+        out, rc = client1.exec_command(
+            sudo=True, cmd="ceph config set mon mon_allow_pool_delete true "
+        )
+        out, rc = client1.exec_command(
+            sudo=True, cmd="ceph fs volume rm cephfs_new --yes-i-really-mean-it "
+        )
+        if client1.node.exit_status == 0:
+            results.append("Volume deleted successfully")
+
+        log.info("Execution of testcase %s ended" % tc1)
+        if client1.node.exit_status == 0:
+            results.append("TC %s passed" % tc1)
+
+        print("Testcase Results:")
+        for res in results:
+            print(res)
+
+        print("Script execution time:------")
+        stop = timeit.default_timer()
+        total_time = stop - start
+        mins, secs = divmod(total_time, 60)
+        hours, mins = divmod(mins, 60)
+        print("Hours:%d Minutes:%d Seconds:%f" % (hours, mins, secs))
```

Comment:
As the test execution duration is already captured higher up... do you want to do this here?